src/CONST.ts

```diff
@@ -1889,6 +1890,12 @@ const CONST = {
         COMPACT: 'compact',
         DEFAULT: 'default',
     },
+    SUBSCRIPTION: {
+        TYPE: {
+            ANNUAL: 'yearly2018',
+            PAYPERUSE: 'monthly2018',
```

Comment:
Is there a specific reason for the `2018` suffixes in these constants?

Reply:
I'm not sure, it's been documented like that in the doc. I think @amyevans could provide more info

Reply:
I think this is a specific start year for subscription

Reply:
It's legacy naming, 2018 was when we updated our pricing structure, but for consistency with other areas of our code we should just keep it 😄